modules/openapi-generator/src/main/resources/typescript-nestjs/package.mustache

```diff
@@ -28,9 +28,11 @@
     "@nestjs/common": "^{{nestVersion}}",
     "@nestjs/core": "^{{nestVersion}}",
     "@nestjs/platform-express": "^{{nestVersion}}",
+    "@nestjs/config": "1.1.6",
```

Comment:
We should add a variable for setting the config version, some of our applications are using 1.2.0 already